pkg/apis/kudo/v1alpha1/instance_types.go

```diff
@@ -109,3 +416,13 @@ type InstanceList struct {
 func init() {
 	SchemeBuilder.Register(&Instance{}, &InstanceList{})
 }
+
+// InstanceError indicates error on that can also emit a kubernetes warn event
+type InstanceError struct {
```

Comment:
this is great... I would really like a function for creation... similar to `errors.New(`
this something like `instance.Error(`